src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -239,6 +241,10 @@ private function parseConfigs(\DOMElement $node, $path)
             $defaults['_controller'] = $controller;
         }
 
+        if ($redirectTo = $node->getAttribute('redirect-to')) {
+            $defaults += $this->redirectToDefaults($redirectTo, $node->getAttribute('redirect-permanent'));
```

Comment:
not good. The attribute value will always be a string. It must be parsed into an actual boolean (see the values accepted by `parseDefaultNode` for the `<bool>` tag)

Reply:
But.. it's working well... not sure `getAttribute` actually returns only `string`s 🤔